src/pages/[platform]/build-a-backend/add-aws-services/rest-api/update-data/index.mdx

```diff
@@ -0,0 +1,208 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Update data',
+  description: 'Using Post, Put, etc. in Amplify',
+  platforms: [
+    'javascript',
+    'react-native',
+    'flutter',
+    'swift',
+    'android',
+    'angular',
+    'nextjs',
+    'react',
+    'vue'
+  ],
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+<InlineFilter filters={['javascript', "angular", "react", "vue", "react-native", "nextjs"]}>
+## PUT data
```

Comment:
why are the PUT headings different across platforms? is this leftover from gen1 docs?